components/blog/PostList.tsx

```diff
@@ -0,0 +1,120 @@
+import Image from "next/image";
+import Link from "next/link";
+import { getStrapiMedia, formatDate } from "../../utils/blog/api-helpers";
+
+export interface Article {
+	id: 4;
+	attributes: {
+		title: string;
+		description: string;
+		slug: string;
+		createdAt: string;
+		updatedAt: string;
+		publishedAt: string;
+		cover: {
+			data: {
+				attributes: {
+					url: string;
+				};
+			};
+		};
+		category: {
+			data: {
+				attributes: {
+					name: string;
+					slug: string;
+				};
+			};
+		};
+		authorsBio: {
+			data: {
+				attributes: {
+					name: string;
+					avatar: {
+						data: {
+							attributes: {
+								url: string;
+							};
+						};
+					};
+				};
+			};
+		};
+	};
+}
+
+interface PostListProps {
+	data: Article[];
+	children?: React.ReactNode;
+}
+const PostList = ({
+	data: articles,
+	children,
+}: PostListProps) => {
+	return (
+		<section className="container p-6 mx-auto space-y-6 sm:space-y-12">
+			<div className="grid justify-center grid-cols-1 gap-6 sm:grid-cols-2 lg:grid-cols-3">
+				{articles.map((article) => {
+					const imageUrl = getStrapiMedia(
```

Comment:
`getStrapiMedia` can "throw".
Please add some form of error handling here.

Reply:
instead of throwing, returning null from getStrapiUrl()